CHANGELOG.rst

```diff
@@ -1,9 +1,24 @@
 .. _changelog:
 
+0.33.0
+------
+
+Renku ``0.33.0`` upgrades the PostgreSQL version that is shipped with the project.
```

Comment:
Could we rephrase so that it is more informative? I am not sure if I understand the concept behind this release. Does it refer to the following?  
 _Renku ``0.33.0`` upgrades the PostgreSQL version that is associated with your Renku projects._

Maybe I lack some background/insight on the implications of this upgrade.